eva-server/src/main/java/uk/ac/ebi/eva/server/ws/RegionWSServer.java

```diff
@@ -114,4 +114,21 @@ public QueryResponse getVariantsByRegion(@PathVariable("regionId") String region
     public QueryResponse getVariantsByRegion() {
         return setQueryResponse("");
     }
+
+    @RequestMapping(value = "", method = RequestMethod.GET)
+    @ResponseBody
+//    @ApiOperation(httpMethod = "GET", value = "Retrieves all the variants from region", response = QueryResponse.class)
+    public QueryResponse getChromosomes(@RequestParam(name = "species") String species,
+                                        HttpServletResponse response)
```

Comment:
It seems that the query including studies can take up to 5 seconds. In that case, and considering that the variants should not be concentrated in just few chromosomes, I think that not having studies filter (like it is now) is better